grunttasks/template.js

```diff
@@ -0,0 +1,26 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+module.exports = function (grunt) {
+  'use strict';
+
+  var config = require('../server/config');
+
+  // Add custom delimiters since our javascript has both mustache and underscore delimiters embedded
+  grunt.template.addDelimiters('js-config-delimiters', '{%', '%}');
+
+  grunt.config('template', {
+    scripts: {
+      options: {
+        data: {
+          config: config
```

Comment:
in that case, should i be passing my `staticPath` in via `initConfig` or should i try passing the entire `config` object and then using `'<%= config.get('server.staticPath') %>'` in the individual grunt tasks?

or are we cool with the mild inconsistency?